### PR TITLE
Notification

### PR DESCRIPTION
**Pull request by Zeeson**
Please review my codes: 
1. i worked on contact for subject, endpoint and form input names
2. also on index.js for contact that is coming from contactUs
3. also routes.js for contact and notification routing

### CODE REVIEW

File: controller/contactUs.js [hunk 1]
```diff
@@ -1,5 +1,27 @@
...
+  body('subjec').isString(),
+  body('message').isString(),
+];
+
+
+// get all contacts
+const getAllContacts = (req, res) => {
+  Contacts.find({})
+    .then(contacts => {
+        res.render("notifications", {contacts: contacts})
```
Comment (tonymj76) [inline controller/contactUs.js:16]: why use render here

File: controller/contactUs.js [hunk 2]
```diff
@@ -1,5 +1,27 @@
 const Contacts = require("../models/Contact")
 
+// Intern Validation rules
+const contactValidationRules = () => [
```
Comment (tonymj76) [inline controller/contactUs.js:4]: you declared this and didn't use it in your create contact method

File: controller/contactUs.js [hunk 3]
```diff
@@ -1,5 +1,27 @@
...
+  body('name').isString(),
+  body('email').isEmail(),
+  body('subjec').isString(),
+  body('message').isString(),
+];
+
+
+// get all contacts
+const getAllContacts = (req, res) => {
```
Comment (tonymj76) [inline controller/contactUs.js:13]: this function is not suppose to be in here... check my admin view in controller/view/index

File: routes/routes.js [hunk 4]
```diff
@@ -4,6 +4,10 @@ const {
   homePage, login, contact, logout, createAdmin
 } = require('../controller');
 
+const {
```
Comment (tonymj76) [inline routes/routes.js:7]: where is the validation of the contact?

File: routes/routes.js [hunk 5]
```diff
@@ -22,6 +26,7 @@ router.post('/create', createAdmin);
 
 router.post('/contact-us', contact);
 router.post('/apply', internValidationRules(), internApply);
+router.get('/notifications', getAllContacts)
```
Comment (tonymj76) [inline routes/routes.js:29]: no need of this right here

File: views/pages/notifications.ejs [hunk 6]
```diff
@@ -12,59 +12,22 @@
     <button role="button" type="button" class="clear-selected">Clear Selected</button>
   </div>
 
+  <% contacts.forEach(function(contact){ %>
   <div class="notifications-list">
     <div class="notification unread">
       <div class="info">
         <h4>Notification One</h4>
+        <h4><%= contact.subject %> </h4>
         <span class="time">2 Feb 2020, 12:44AM</span>
```
Comment (tonymj76) [inline views/pages/notifications.ejs:21]: this time should not be static

File: controller/contactUs.js (hunk 1 above)
Comment (Zeeson) [reply controller/contactUs.js:16]: now: res.status(200).json("notifications", {contacts: contacts})

File: controller/contactUs.js (hunk 2 above)
Comment (Zeeson) [reply controller/contactUs.js:4]: removed

File: controller/contactUs.js (hunk 3 above)
Comment (Zeeson) [reply controller/contactUs.js:13]: should it be removed?

**Pull request merged**